pkg/storkctl/clusterpair.go

```diff
@@ -258,18 +267,136 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 				util.CheckErr(err)
 				return
 			}
-			printMsg("Using PX-Service Endpoint of DR cluster to create clusterpair...\n", ioStreams.Out)
-			ip, port, token, err := getClusterPairParams(dFile, dIP, dPort)
+
+			if len(sIP) > 0 || sPort != "9001" || len(dIP) > 0 || dPort != "9001" {
+				return
```

Comment:
Print an error message and return?

Reply:
With `MarkDeprecated`, a message anyway gets printed like following.

```
➜  stork git:(PWX-30578) ✗ bin/linux/storkctl create clusterpair cp-new18 -p s3 -n kube-system --s3-access-key ***** --s3-secre
t-key ***** --s3-region us-east-1 --s3-endpoint ******** --src-kube-file /tmp/dipti5.config --dest-kub
e-file /tmp/dipti7.config --src-ip 10.13.193.170 --dest-ip 10.13.194.128
Flag --src-ip has been deprecated, instead provide --src-ep
Flag --dest-ip has been deprecated, instead provide --dest-ep
```

hence not adding any extra error message.